ac/ac-ranking/src/meta.js

```diff
@@ -2,7 +2,8 @@
 
 const trainConfig = {
   title: 'Which interface is the best?',
-  guidelines: 'Please drag the items in order.',
+  guidelines:
+    '<p>Please drag the items in order.</p><p><img src="/file?name=cjg7q0o6400063i6ivwiot4op" style={{padding: "5" }} width="306" height="146"/><img src="/file?name=cjg7q4md100073i6impu8qvo4" border="2" width="306" height="112"/><img src="/file?name=cjg7q50t600083i6i0spm9jdi" border="2" width="300" height="262"/><img src="/file?name=cjg7q58xy00093i6icte72u16" border="2" width="302" height="98"/></p>',
```

Comment:
This files you uploaded only on your local frog server so they cannot be used with the examplePreview like this.

Reply:
Is there a place where we can permanently put files for FROG or should I just save them somewhere myself?

Reply:
Nevermind, this is a change Stian made and I will fix this once we have the other code merged.